Automattic/woocommerce-payments
# Disable activation of LPM's requiring additional currencies when MC is disabled

Fixes #7154

#### Changes proposed in this Pull Request



This PR implements the disabled checkboxes and tooltips feature for LPM's that require additional currencies that doesn't match with the store currency to operate, when the multi currency feature is turned off. The fix both applies the LPM activation box on Payments > Settings, and the LPM onboarding page.







- The notice displayed while MC is on and the LPM requires another currency to be enabled:

<img width="725" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3295/0e22b47b-08ee-4da5-8dce-85f491cff64a">

- The current screen when MC is off

<img width="733" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3295/082a1c31-1742-40fd-8e9e-eef728304f43">

- Suggested fix

![image](https://github.com/Automattic/woocommerce-payments/assets/3295/75337e4f-bfbf-498c-b01f-39e2650f21eb)

- Suggested fix on the Payments > Settings page
 
<img width="1138" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3295/2ed7ee88-ab84-4f43-8427-5ea98eaf62f1">


#### Testing instructions





* Go to WooCommerce > Settings and change the store currency to "USD" if not selected.
* Go to Payments > Settings > Advanced and uncheck the "Enable Multi-Currency" checkbox.
* Save Changes.
* Navigate to http://localhost:8082/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Fadditional-payment-methods
* See the onboarding page shows the LPM's which require additional currencies as disabled, with the correct tooltip.
* Go to Payments > Settings and see the same effect on the Additional payment methods box too.
* Change the store currency to "EUR" and see that the LPM's are enabled, but the ones requiring currencies other than EUR got disabled on both screens.

-------------------


**Post merge**
